### PR TITLE
View the threads & posts of a profile

### PR DESCRIPTION
**Pull request by Uptaker**

### CODE REVIEW

File: frontend/src/components/Profile/ProfileThreadsTab.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        </Space>
+    );
+
+    return (
+        <List
+            itemLayout="vertical"
+            size="large"
+            pagination={{
+                onChange: page => {
+                    console.log(page);
```
Comment (mariliisaar) [inline frontend/src/components/Profile/ProfileThreadsTab.js:22]: console log :D Also, is something supposed to happen here?

File: frontend/src/components/Profile/ProfileUserContent.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        tab: 'Threads',
+    },
+    {
+        key: 'posts',
+        tab: 'Posts',
+    },
+];
+
+const ProfileUserContent = ({username, posts, threads, displayName}) => {
+    const [activeTabKey, setActiveTabKey] = useState('tab1');
```
Comment (mariliisaar) [inline frontend/src/components/Profile/ProfileUserContent.js:18]: I'm probably a bit confused but shouldn't we set this initially to either threads or posts?

**Review by mariliisaar — APPROVED** [review]
One question :)

File: frontend/src/components/Profile/ProfileUserContent.js (hunk 2 above)
Comment (Uptaker) [reply frontend/src/components/Profile/ProfileUserContent.js:18]: I didn't actually want to display all the threads/posts but let the user decide if they want to see those

File: frontend/src/components/Profile/ProfileThreadsTab.js (hunk 1 above)
Comment (Uptaker) [reply frontend/src/components/Profile/ProfileThreadsTab.js:22]: That's left over from ant design's template and it just logs what page you're on :P

**Pull request merged**